template/helper/listbox.php

```diff
@@ -96,13 +96,11 @@ public function tags($config = array())
                 var form = $(element[0].form);
                 
                 form.submit(function() {
-                    if (!element.val()) {
```

Comment:
@jebbdomingo This code was added for the same purpose. Did you check what the value of `element.val()` was when it's empty?

Reply:
@ercanozkaya Thanks for pointing out. I pushed a fix, could you please check it out? Thanks.